website/integrations/services/aws/index.md

```diff
@@ -71,80 +70,78 @@ return [
 
 The RoleSessionMapping specifies what identifier will be shown at the top of the Management Console ([see](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_saml_assertions.html#saml_role-session-attribute)).
 
-This mapping needs to have the SAML Name field set to "https://aws.amazon.com/SAML/Attributes/RoleSessionName".
+This mapping needs to have the SAML Name field set to `https://aws.amazon.com/SAML/Attributes/RoleSessionName`.
 
 To use the user's username, use this snippet
 
 ```python
 return user.username
 ```
 
-# Method 2: IAM Identity Center
+## Method 2: IAM Identity Center
 
-## Preparation
+### Preparation
 
--   A certificate to sign SAML assertions is required. You can use authentik's default certificate, or provide/generate one yourself.
--   You may pre-create an AWS application.
+- A certificate to sign SAML assertions is required. You can use authentik's default certificate, or provide/generate one yourself.
+- You may pre-create an AWS application.
 
-## How to integrate with AWS
+### How to integrate with AWS
 
 In AWS:
 
--   In AWS navigate to: _IAM Identity Center_ -> _Settings_ -> _Identity Source (tab)_
--   On the right side click _Actions_ -> _Change identity source_
--   Select _External Identity Provider_
--   Under _Service Provider metadata_ download the metadata file.
+- In AWS, navigate to: **IAM Identity Center** -> **Settings** -> **Identity Source (tab)**
```

Comment:
We can use the bold marking around the entire path, like this, to make make it easier: 
**IAM Identity Center -> Settings -> Identity Source (tab)**